pkg/server/commands/list_objects.go

```diff
@@ -58,21 +57,18 @@ func (q *ListObjectsQuery) handler(
 	targetObjectType := req.GetType()
 	targetRelation := req.GetRelation()
 
-	model, err := q.Datastore.ReadAuthorizationModel(ctx, req.GetStoreId(), req.GetAuthorizationModelId())
-	if err != nil {
-		if errors.Is(err, storage.ErrNotFound) {
-			return serverErrors.AuthorizationModelNotFound(req.GetAuthorizationModelId())
-		}
-		return err
+	typesys, ok := typesystem.TypesystemFromContext(ctx)
+	if !ok {
+		panic("typesystem missing in context")
```

Comment:
this should not be a panic since we can recover from this error by actually building the type system